esp/bindings/http/platform/httpbinding.cpp

```diff
@@ -271,6 +270,110 @@ EspHttpBinding::EspHttpBinding(IPropertyTree* tree, const char *bindname, const
             memCachedInitString.set("--SERVER=127.0.0.1");//using local memcached server
     }
 #endif
+    if (!m_secmgr.get())
+        return;
+
+    processName.set(procname);
+    const char* authDomain = bnd_cfg->queryProp("@authDomain");
+    if (!isEmptyString(authDomain))
+        domainName.set(authDomain);
+    else
+        domainName.set("default");
+
+    setSDSSession();
+    readAuthDomainCfg(proc_cfg);
+
+    checkSessionTimeoutSeconds = proc_cfg->getPropInt("@checkSessionTimeoutSeconds", -1);
+    if (checkSessionTimeoutSeconds == -1)
```

Comment:
Do you mean that users will put -1 in the config to mean "default"?  Otherwise you could just do one line:

checkSessionTimeoutSeconds = proc_cfg->getPropInt("@checkSessionTimeoutSeconds", ESP_CHECK_SESSION_TIMEOUT);